source/speech.py

```diff
@@ -436,8 +441,16 @@ def getIndentationSpeech(indentation):
 			res.append(symbol)
 		else:
 			res.append(u"{count} {symbol}".format(count=count, symbol=symbol))
-
-	return " ".join(res)
+		quarterTones += (count*4 if raw[0]== "\t" else count)

```

Comment:
Out of interest, why the `[0]`? At this point, we should already be dealing with single characters... and even if we weren't in some future/alternative universe, that suggests the space is a double byte character or something, so the second character could be important.